build.js

```diff
@@ -39,11 +41,14 @@ for (const [key, value] of Object.entries(entries)) {
         // 'production' in all other cases.
         process && process.env && process.env.NODE_ENV === "development"
           ? '"development"'
-          : '"production"',
-    },
+          : '"production"'
+    }
   };
 
-  esbuild.build(options).then((result) => {
-    console.log("Built " + key + ".js");
-  });
+  esbuild
+    .build(options)
+    .then((result) => {
+      console.log("Built " + key + ".js");
+    })
+    .catch(() => process.exit(1));
```

Comment:
Ah hah! @anthonydillon @ClementChaumel it turns out there was a build error, but `esbuild` didn't report anything so I've changed it to fail the build if it has an issue.

I've resolved the issues with this branch so it should be good for a review/QA now. I'll also propose the error catching against master.